src/groupeddataframe/grouping.jl

```diff
@@ -435,7 +435,16 @@ of `combine(map(f, groupby(df, cols)))`.
 function combine(f::Any, gd::GroupedDataFrame)
     if length(gd) > 0
         idx, valscat = _combine(f, gd)
-        return hcat!(gd.parent[idx, gd.cols], valscat, makeunique=true)
+        keys = _names(gd.parent)[gd.cols]
+        for key in keys
```

Comment:
a small gripe `keys` is a standard function name from Base. It is not a big problem though - please decide if you want to rename it or not.

Reply:
I'd say that's not a problem, especially given that the function is quite short.